packages/prisma/schema.prisma

```diff
@@ -46,6 +46,7 @@ model EventType {
   teamId                  Int?
   bookings                Booking[]
   availability            Availability[]
+  webhooks                Webhook[]
```

Comment:
These changes should have trigger changes inside `packages/prisma/zod` Let's commit those as well.

Reply:
Okay, got it